verify/assets/index.html

```diff
@@ -116,153 +155,161 @@ <h2>Twilio Verify</h2>
     const phoneInput = window.intlTelInput(phoneInputField, {
       // https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
       preferredCountries: ["us", "co", "in", "de"],
-      separateDialCode: true,
       utilsScript:
-        "https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/16.0.11/js/utils.js"
+        "https://cdn.jsdelivr.net/npm/intl-tel-input@17.0.13/build/js/utils.js",
     });
 
-    $("#channel-select").change(function() {
-      const channel = $("input[name='channel']:checked").val();
-      if (channel === "email") {
-        $(".phone-input").hide();
-        $(".email-input").show();
-      } else if (channel === "sms" || channel === "call") {
-        $(".phone-input").show();
-        $(".email-input").hide();
-      }
-    });
+    function selectedChannel() {
+      const checked = "input[name='channel']:checked";
+      return document.querySelector(checked).value;
+    }
+
+    function showEmailInput() {
+      document.getElementById("phone-input").style.display = "none";
+      document.getElementById("email-input").style.display = "block";
+    }
+
+    function showPhoneNumberInput() {
+      document.getElementById("phone-input").style.display = "block";
+      document.getElementById("email-input").style.display = "none";
+    }
+
+    function showModalStatus(message, color = "gray") {
+      let status = document.getElementById("modal-status");
+      status.style.color = color;
+      status.textContent = message;
+    }
+
+    function showError(error) {
+      console.error(error);
+      showStatus(error, (color = "#a94442"));
+    }
 
-    function showVerificationStatus(alertType, message) {
-      var content = $(".result-message");
-      content.empty();
-      content.append($("<div>")
-        .addClass(`alert alert-${alertType}`)
-        .attr("role", "alert")
-        .text(message))
+    function showStatus(message, color = "gray") {
+      let status = document.getElementById("status");
+      status.style.color = color;
+      status.textContent = message;
     }
 
-    function showVerificationStartError(error) {
-      var content = $("#form-error");
-      content.empty();
-      content.append(`Error: ${error}`);
+    function clearStatus() {
+      document.getElementById("status").textContent = "";
+      document.getElementById("modal-status").textContent = "";
     }
 
+    const modal = document.getElementById("otp-modal");
```

Comment:
Sometimes you cache your DOM elements before they are used and refer to the pointer, like this. Others you make requests to `document.getElementById` each time a function is run. It's more consistent to choose one approach and, while it doesn't matter so much in this example, it would be more performant to cache the elements.